database/nosql/cassandra/egg-apache-cassandra.json

```diff
@@ -0,0 +1,60 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-08-03T22:11:40+02:00",
+    "name": "Apache Cassandra",
+    "author": "p.zarrad@outlook.de",
+    "description": "Apache Cassandra is an open source NoSQL distributed database trusted by thousands of companies for scalability and high availability without compromising performance. Linear scalability and proven fault-tolerance on commodity hardware or cloud infrastructure make it the perfect platform for mission-critical data.",
+    "features": null,
+    "images": [
+        "ghcr.io\/parkervcp\/yolks:cassandra_java11_python3",
+        "ghcr.io\/parkervcp\/yolks:cassandra_java8_python2"
+    ],
+    "file_denylist": [],
+    "startup": ".\/start.sh",
```

Comment:
I definitely don't agree with using start scripts. I know you want to validate not using default passwords and things but I want to avoid them at all costs.

Reply:
@parkervcp Totally forgot about this one.

To start the server and have a shell, starting Cassandra and CQLSH will be necessary.
I'd suggest to solve this by doing it the way the ARK image does it. This will at least eliminate the user editable script.
When dropping the password check and only keeping necessary code, this would shorten the script by a significant margin.

See: https://github.com/parkervcp/eggs/blob/17a283334ae5737720ab52e95cd9ba4c7d1b680c/game_eggs/steamcmd_servers/ark_survival_evolved/egg-ark--survival-evolved.json#L18

Is this an appropriate solution?

Reply:
That's is probably a better approach.